resources/lib/TorecSubtitlesDownloader.py

```diff
@@ -267,13 +285,16 @@ def _confirm_download_code(self, sub_id, option_id):
         response      = self.opener.open("%s/ajax/sub/t7/guest_dl_code.asp" % self.BASE_URL, urllib.urlencode(params))
         response_data = response.read()
 
-    def _try_get_download_link(self, sub_id, option_id, guest_token):
+    def _try_get_download_link(self, sub_id, option_id, guest_token, is_login=False):
+        if guest_token is None:
```

Comment:
To be able to get `guest_token` both as empty string and None, if getting None I am converting to empty string. I need it to be empty string to send to Torec.